x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/mapper/SemanticTextFieldMapperTests.java

```diff
@@ -253,14 +282,14 @@ public void testInvalidInferenceEndpoints() {
         {
             Exception e = expectThrows(
                 MapperParsingException.class,
-                () -> createMapperService(fieldMapping(b -> b.field("type", "semantic_text").field(INFERENCE_ID_FIELD, "")))
+                () -> createMapperService(fieldMapping(b -> b.field("type", "semantic_text").field(INFERENCE_ID_FIELD, "")), false)
```

Comment:
We should use `useLegacyFormat` here

Reply:
Done in https://github.com/elastic/elasticsearch/pull/119183/commits/05458c6b661e2ae2b172e17b62e906f993cfbd26